docs/tutorial/part-three/index.md

```diff
@@ -4,54 +4,54 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to part three!
+Bem-vindo a parte três!
 
-## What's in this tutorial?
+## O que veremos nesse tutorial?
 
-In this part, you'll learn about Gatsby plugins and creating "layout" components.
+Nessa parte, você irá aprender sobre plugins do Gatsby e como criar componentes de layout.
 
-Gatsby plugins are JavaScript packages that help add functionality to a Gatsby site. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
+Plugins do Gatsby são pacotes JavaScript que ajudam a adicionar funcionalidades a um site Gatsby. O Gatsby foi projetado para ser extensível, o que significa que os plugins podem estender e modificar praticamente qualquer coisa que o Gatsby faz.
 
-Layout components are for sections of your site that you want to share across multiple pages. For example, sites will commonly have a layout component with a shared header and footer. Other common things to add to layouts are a sidebar and/or navigation menu. On this page for example, the header at the top is part of gatsbyjs.org's layout component.
+Componentes de layout são para seções do seu site que você quer compartilhar entre diversas páginas. Por exemplo, normalmente sites tem componentes de layout para um cabeçalho e rodapé que são compartilhados. Outras coisas comuns de se adicionar em um layout são *sidebars* e/ou menu de navegação. Nesta página, por exemplo, o cabeçalho no topo é parte de um componente de layout do gatsbyjs.org.
```

Comment:
Para manter o padrão do restante das documentações, sugiro que deixe em itálico as palavras não traduzidas do inglês.

```suggestion
Componentes de layout são para seções do seu site que você quer compartilhar entre diversas páginas. Por exemplo, normalmente sites tem componentes de layout para um cabeçalho e rodapé que são compartilhados. Outras coisas comuns de se adicionar em um layout são _sidebars_ e/ou menu de navegação. Nesta página, por exemplo, o cabeçalho no topo é parte de um componente de layout do gatsbyjs.org.
```